src/main/java/com/autotune/database/table/KruizeMetadataEntry.java

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.autotune.database.table;
+
+import jakarta.persistence.*;
+
+/**
+ * This is a Java class named KruizeMetadataEntry annotated with JPA annotations.
+ * It represents a table named kruize_metadata in a relational database.
+ * <p>
+ * The class has the following fields:
+ * <p>
+ * id: A unique identifier for each experiment detail.
+ * version: A String representing the version of the experiment.
+ * clusterGroupName: A String representing the datasource name.
+ * clusterName: A String representing the cluster name.
+ * namespace: A String representing the namespace name.
+ * workloadType: A String representing the type of workload.
+ * workloadName: A String representing the workload name.
+ * containerName: A String representing the container name.
+ * containerImageName: A String representing the container image name.
+ */
+@Entity
+@Table(name = "kruize_metadata")
+public class KruizeMetadataEntry {
```

Comment:
This should be
```
@Table(name = "kruize_dsmetadata")
public class KruizeDSMetadataEntry {
```
Metadata is too generic and broad

Reply:
Got it. Updated the name to 'KruizeDSMetadataEntry'